shoop/core/fields/__init__.py

```diff
@@ -52,12 +53,17 @@ def deconstruct(self):
         return (name, path, args, kwargs)
 
 
-class MoneyField(models.DecimalField):
+class CurrencyField(models.CharField):
```

Comment:
Sure, no worries. As mentioned, it can be done later, and adding a `formfield` method to a class won't cause new migrations.